src/tests/ftest/deployment/disk_failure.py

```diff
@@ -0,0 +1,186 @@
+                                                    "test": self.ior_test_sequence[0],
+                                                    "fail_on_warning": False}))
+            # Launch the IOR threads
+            for thrd in threads:
+                self.log.info("Thread : %s", thrd)
+                thrd.start()
+                time.sleep(5)
+
+            count = 0
+            # Evict some of the targets from the system
+            for key, _ in device_info.items():
+                if count < 1:
+                    resp = self.set_nvme_faulty(device_info[key]["uuid"])
+                    time.sleep(5)
+                    self.log.info(resp)
+                count = count + 1
+            done = "Completed setting all devices to fault"
+            self.print_and_assert_on_rebuild_failure(done)
+            for thrd in threads:
+                thrd.join()
+
+            # Now replace the faulty NVME device.
+            count = 0
+            for key, _ in device_info.items():
+                if count < 1:
+                    resp = self.dmg_command.storage_replace_nvme(old_uuid=device_info[key]["uuid"],
+                                                                 new_uuid=device_info[key]["uuid"])
+                    time.sleep(10)
+                    self.log.info(resp)
+                    # Convert target list to string
+                    targets = ",".join(map(str, device_info[key]["tgts"]))
+                    # Now reintegrate the target to appropriate rank.
+                    output = self.dmg_command.pool_reintegrate(self.pool.uuid,
+                                                               device_info[key]["rank"],
+                                                               targets)
+                    time.sleep(15)
+                count = count + 1
+            done = "Faulty NVMEs replaced"
+            self.print_and_assert_on_rebuild_failure(output)
+            self.log.info(done)
+
+        # After completing the test, check for container integrity
+        for val in range(0, num_pool):
+            display_string = "Pool{} space at the End".format(val)
+            self.pool = pool[val]
+            self.pool.display_pool_daos_space(display_string)
+            self.run_ior_thread("Read", oclass=self.ior_cmd.dfs_oclass.value,
+                                test=self.ior_test_sequence[0])
+            self.container = self.pool_cont_dict[self.pool][0]
+            kwargs = {"pool": self.pool.uuid,
+                      "cont": self.container.uuid}
+            output = self.daos_command.container_check(**kwargs)
+            self.log.info(output)
+
+    def test_disk_failure_w_rf(self):
+        """Jira ID: DAOS-11284
+        Test disk failures during the IO operation.
+
+        :avocado: tags=all,manual
+        :avocado: tags=hw,medium
+        :avocado: tags=deployment
+        :avocado: tags=disk_failure
```

Comment:
Can we please use the test method name as the unique tag? For this and other tests
```suggestion
        :avocado: tags=disk_failure,test_disk_failure_w_rf
```